waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -106,9 +108,16 @@
 
   public @Rule ServerSocketRule graphite = new ServerSocketRule();
   public @Rule TemporaryFolder temporaryFolder = new TemporaryFolder();
-  public @Rule ThriftHiveMetaStoreJUnitRule localServer = new ThriftHiveMetaStoreJUnitRule(LOCAL_DATABASE);
-  public @Rule ThriftHiveMetaStoreJUnitRule remoteServer = new ThriftHiveMetaStoreJUnitRule(REMOTE_DATABASE);
-  public @Rule ThriftHiveMetaStoreJUnitRule newRemoteServer = new ThriftHiveMetaStoreJUnitRule();
+  public static Map<String,String> hiveConfig = new HashMap<>();
```

Comment:
With the move to BeeJU 5.x which supports Hive-3.x I'm not sure we need to do this, it should be doing this itself?